epochStart/bootstrap/disabled/disabledCurrentNetworkEpochProviderHandler.go

```diff
@@ -0,0 +1,33 @@
+package disabled
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/data"
+)
+
+// CurrentNetworkEpochProviderHandler -
```

Comment:
unexported and constructor?
:+1: for not adding `disabled` in the component's name because it is "fetched" from the package name

Reply:
done